.github/workflows/check.yml

```diff
@@ -0,0 +1,22 @@
+name: CI Check
+
+on: [pull_request]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
```

Comment:
Why do we need this?

Reply:
Don't we have some ITs that use docker?
In that case it is generally advisable to login to dockerhub because of the pull rate limiting.